solidity/test/LiquidityProtection.js

```diff
@@ -680,6 +684,38 @@ describe('LiquidityProtection', () => {
                 expect(newBalance).to.be.bignumber.equal(new BN(0));
             });
 
+            for (let x = 24; x < 30; x++) {
```

Comment:
Great tests! 💪

Reply:
TX.
Should probably be a lot more thorough, with two more loop scopes - one for the initial balance of the pool in the base token, and one for the initial balance of the pool in the network token.
Unfortunately, the way that our `beforeEach` clause and `initPool` functions are written, it's quite a "heavy" task to change those initial balances (unless we do the whole thing in a separate test file).
In addition to all of the above, it would be useful to choose to use in those loops various amounts which are not necessarily divisible by each other. If the `poolAvailableSpace` function is called under some configuration of the pool, and misses by a single wei, a user transaction based on its return value will revert.